recipe-box-backend/src/main/java/com/endava/recipebox/repository/UserIngredientRepository.java

```diff
@@ -0,0 +1,7 @@
+package com.endava.recipebox.repository;
+
+import com.endava.recipebox.model.UserIngredient;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface UserIngredientRepository extends JpaRepository<UserIngredient, Long> {
```

Comment:
the ingredients are specific per user? Why do we need to have UserIngredient instead of IngredientRepository